app/session.ts

```diff
@@ -1,105 +1,139 @@
-    'ecx-cors', // warning(hugo) minify all keys
-    'bx-dynamic-user-agent',
-    'ecx-api-handler',
-  ];
+// const orderListeners = (listeners: any) => {
+//   const orderableOrigins = [
+//     'ecx-cors', // warning(hugo) minify all keys
+//     'bx-dynamic-user-agent',
+//     'ecx-api-handler',
+//   ];
 
-  const orderedListeners = orderableOrigins.reduce(
-    (orderedList: any[], origin: string) => {
-      const listener = listeners.find(
-        (l: any) => l.context.origin && l.context.origin === origin
-      );
+//   const orderedListeners = orderableOrigins.reduce(
+//     (orderedList: any[], origin: string) => {
+//       const listener = listeners.find(
+//         (l: any) => l.context.origin && l.context.origin === origin
+//       );
 
-      if (listener) {
-        orderedList.push(listener);
+//       if (listener) {
+//         orderedList.push(listener);
 
-        return orderedList;
-      }
+//         return orderedList;
+//       }
 
-      return orderedList;
-    },
-    []
-  );
+//       return orderedList;
+//     },
+//     []
+//   );
 
-  const unorderedListeners = listeners.filter(
-    (l: any) => !orderableOrigins.includes(l.context.origin)
-  );
+//   const unorderedListeners = listeners.filter(
+//     (l: any) => !orderableOrigins.includes(l.context.origin)
+//   );
 
-  return [...unorderedListeners, ...orderedListeners];
-};
+//   return [...unorderedListeners, ...orderedListeners];
+// };
 
-const webRequestHandler = (listeners: any) => {
-  const sortedListeners = orderListeners(listeners);
+// const webRequestHandler = (listeners: any) => {
+//   const sortedListeners = orderListeners(listeners);
 
-  const response = sortedListeners.reduce(
-    async (accumulator: any, element: any) => {
-      if (accumulator.cancel) {
-        return accumulator;
-      }
+//   const response = sortedListeners.reduce(
+//     async (accumulator: any, element: any) => {
+//       if (accumulator.cancel) {
+//         return accumulator;
+//       }
 
-      const result = await element.apply();
+//       const result = await element.apply();
 
-      return { ...accumulator, ...result };
-    },
-    { cancel: false }
-  );
+//       return { ...accumulator, ...result };
+//     },
+//     { cancel: false }
+//   );
 
-  return response;
-};
+//   return response;
+// };
 
-const callbackMethods = [
-  'onBeforeRequest',
-  'onBeforeSendHeaders',
-  'onHeadersReceived',
-];
+// const callbackMethods = [
+//   'onBeforeRequest',
+//   'onBeforeSendHeaders',
+//   'onHeadersReceived',
+// ];
+
+const defaultUserAgent = [
+  'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_1)',
+  'AppleWebKit/537.36 (KHTML, like Gecko)',
+  'Chrome/87.0.4280.141',
+  'Safari/537.36',
+].join(' ');
 
 const whatsappUserAgent = [
   'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_1)',
   'AppleWebKit/537.36 (KHTML, like Gecko)',
-  'Chrome/76.0.3809.100 Safari/537.36',
+  'Chrome/76.0.3809.100',
+  'Safari/537.36',
 ].join(' ');
```

Comment:
With the new default UA, this one is probably no relevant anymore

Reply:
It works, so I decided not to change it.

Reply:
Good for me :)